common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -35,10 +37,22 @@
  */
 public final class InternalThreadLocalMap extends UnpaddedInternalThreadLocalMap {
 
+    private static final InternalLogger logger = InternalLoggerFactory.getInstance(InternalThreadLocalMap.class);
+
     private static final int DEFAULT_ARRAY_LIST_INITIAL_CAPACITY = 8;
+    private static final int STRING_BUILDER_INITIAL_SIZE;
+    private static final int STRING_BUILDER_MAX_SIZE;
 
     public static final Object UNSET = new Object();
 
+    static {
+        STRING_BUILDER_INITIAL_SIZE = SystemPropertyUtil.getInt("io.netty.threadLocalStringBuilderInitialSize", 1024);
```

Comment:
what about breaking out the name space more clearly: `io.netty.threadLocalMap.stringBuilder.initialSize` ... @normanmaurer - wdyt?